### DISCUSSION TITLE
feat(secret): add support of Docker registry credentials

### DISCUSSION THREAD

**Discussion by bgoareguer** [aquasecurity/trivy#5664] [Ideas]
### Description

[This](https://blog.aquasec.com/the-ticking-supply-chain-attack-bomb-of-exposed-kubernetes-secrets) blog post shows how to detect Docker registry credentials with Trivy.

I suggest to add the suggested detection rule to the default set of rules in the Trivy code base.

### Target

None

### Scanner

Secret

**Comment by itaysk** [comment]
Thanks for reporting. To clarify, this is the suggested rule in the blog: https://gist.github.com/yakirk/f0d5a4cb3893914f956336a9edd32cfe

**Comment by knqyf263** [comment]
Created https://github.com/aquasecurity/trivy/issues/5669
